### ISSUE TITLE
Server 2012r2

### ISSUE THREAD

**Issue by maci0** [vrozenfe/qxl-dod#1]
Does the win 8.1 target work for server 2012 R2 too ?

**Comment by vrozenfe** [comment]
Yes, it should work on WS 2012 R2, and the driver is also signed for
this platform.

**Comment by sonicflare** [comment]
I can confirm that it does work on Windows Server 2012 R2.
Have not tested it without Test Mode.

**Comment by CedrX** [comment]
I have a problem with windows server 2012 R2. I got crashes with kernel power management of windows. I tried to disable powermanagerment in config panel but it still crash for the same reason :-(

**Comment by vrozenfe** [comment]
What is the BSOD bugcheck (error) code?
Thanks,
Vadim.

**Comment by CedrX** [comment]
Oops. I think I made a mistake. After investigation I'm now sure that the crashes were not due to the qxl driver ...
It works perfectly with my virtualized windows server 2012.
Sorry.
